File: RSMK2/RSMK2/Provider.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RSMK2
{
    [Serializable]
    public class Provider
    {
        private int _cooperation;
        private string _name;
        private string _material;

        public int GetCooperation() => _cooperation;
        public void SetCooperation(int cooperation) { _cooperation = cooperation; }
        public string GetName() => _name;
        public void SetName(string name) { _name = name; }
        public string GetMaterial() => _material;
        public void SetMaterial(string material) { _material = material; }

        public Provider() { }

        public Provider(int cooperation, string name, string material)
        {
            _cooperation = cooperation;
            _name = name;
            _material = material;
        }
    }
}

File: zwsserver/src/types/Provider.java
package types;

public class Provider {
    private int cooperation;
    private String name;
    private String product;

    public Provider() {
    }

    public Provider(int cooperation, String name, String product) {
        this.cooperation = cooperation;
        this.name = name;
        this.product = product;
    }

    public int getCooperation() {
        return cooperation;
    }

    public void setCooperation(int cooperation) {
        this.cooperation = cooperation;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getProduct() {
        return product;
    }

    public void setProduct(String product) {
        this.product = product;
    }
}
